src/components/item-display-panel/item-display-panel.component.ts

```diff
@@ -31,7 +31,91 @@ export class ItemDisplayPanelComponent {
         this.onChangeCallback(this._visible);  
     }
 
+    @Input()
+    get boxShadow() {
+        let value: any = this._boxShadow;
```

Comment:
The type conversion should be handled on set as this will prevent it from ever being the incorrect type in this class, for example:

![image](https://user-images.githubusercontent.com/20795331/28024602-19fe33fa-6589-11e7-9650-34423930e23e.png)